django_project/feti/static/feti/js/feti-backbone/feti-route.js

```diff
@@ -6,19 +6,20 @@ var AppRouter = Backbone.Router.extend({
     routes: {
         "": "landing_page",
         "/login": "login_page",
-        "/map/:mode": "show_map",
+        "map/:mode": "show_map",
```

Comment:
@cchristelis it doesn't work with leading slash. I think I should remove the others.
The router example : http://backbonejs.org/#Router